modules/ROOT/pages/getting-started-tutorial.adoc

```diff
@@ -0,0 +1,307 @@
+:experimental:
+= Getting Hands-on with Fedora CoreOS
+
+== Prerequisites:
+
+- You have basic knowledge of what Fedora CoreOS is.
+- You have libvirtd installed and active on out system.
+- This guide is written targeting a Linux environment with a working `libvirt / kvm` setup.
+
+== Let's get Started:
+
+First of all, We would have to download an Official Stable Virtual QEMU xz archive [.qcow.xz], which you can download from https://getfedora.org/en/coreos/download?tab=metal_virtualized&stream=stable[here].
+
+We've downloaded the archive, Now we would have to verify it by clicking on `Verify signature and SHA256` and Download the checksum file and signature and placing both files in the same directory as the Disk image and follow the instructions mentioned there to ensure the integrity of the archive.
+
+Once the archive is verified, We can extract it with:
+
+[source,bash]
+----
+xz -d [archive_name]
+----
+
+Along with the extracted `.qcow2` image, you would also need two other programs in order to transpile and validate the ignition scripts, which are:
+
+- fcct : Fedora CoreOS Config Transpiler.
+- ignition-validate : A tool to validate a config for ignition.
+
+You can do so by using your favourite package manager, If you're using Fedora as your Host Operating System.
+You can install it by using dnf:
+
+[source,bash]
+----
+dnf install -y fcct ignition-validate
+----
+
+If these are not packaged by your distribution, you can use them in a container by using either podman or docker.
+You can find them here: https://quay.io/repository/coreos/fcct[fcct] and https://quay.io/repository/coreos/ignition-validate[ignition-validate]
+
+You can use them as following:
+
+[source,bash]
+----
+podman/docker run --rm -i quay.io/coreos/ignition-validate - < myconfig.ign
+----
+
+Or get the Binaries and Source here, https://github.com/coreos/fcct/releases[fcct] and https://github.com/coreos/ignition/releases[ignition-validate]
+
+Once you have them, we can get started with writing our first Ignition file.
+
+== What is Ignition:
+
+Ignition files are used to customize a delivered artifact which is a disk image in our case.
+
+Fedora CoreOS uses Ignition to provision a node in an automated fashion. Ignition config files are written in JSON and typically not user friendly.
+For that reason Fedora CoreOS offers the Fedora CoreOS Config Transpiler (also known as FCCT) that will create Ignition configs from a more user friendly format.
+Additionally we offer the ignition-validate sub-utility that can be used to verify Ignition config and catch issues before launching a machine.
```

Comment:
Suggestion: Additionally the ignition-validate tool can be used to verify Ignition configs and catch issues before launching a machine.

Reply:
fixed: https://github.com/coreos/fedora-coreos-docs/pull/134/commits/f9693ca169653abb2cb8bc806f1426b006718602